// // import { EventEmitter } from "events";
// // interface IThing { }
// // const siblings: EventEmitter[] = []
// // interface ITransport extends IThing { }
// // class Transport extends EventEmitter implements ITransport {
// //   maxSpeed: number[] = [1]
// //   go(to: { x: number, y: number }): Promise<number> {
// //     siblings.forEach(c => {
// //       if (this.maxSpeed.find(ms => ms === 2)) {
// //         this.emit('run')
// //       }
// //     })
// //     return Promise.resolve(1)
// //   }
// // }

// const args = {
//   help: null
// }
// const result6464: Date = func6785(args)/*
// if I stand on error in func6785 ("code": "2304",	"message": "Cannot find name 'func6785'." I expect to be suggested with "declare function func6785" 

// which will generate the following: 
// */

// const arr: number[][] = literal67.method34(1, ['0'], { r: /12/g }) /* 
// If I stand over error in literal67 ("code": "2304"   "message": "Cannot find name 'literal67'.",)

// I expect to be suggested with "declare object 'literal67'" to generate the following statement right before this one : 

// const literal76 = {
// method34: (arg1: number, arg2: string[], arg3: {r: RegExp})=>{return null / or throw not impl}
// }

// */

// a123.foo({d: [1,2]})

// var iiii98= sdfd888(1,[])

// pepe7 = {a: 3, f: []}
